File: Python Exams/Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/3.CoffeMachine/3.CoffeMachine.py
coffee = input()
ingredients = input()
count = int(input())

items_dict ={
    'Espresso':{'Without': 0.90, 'Normal': 1.00, 'Extra': 1.20 },
    'Cappuccino':{'Without': 1.00, 'Normal': 1.20, 'Extra': 1.60 },
    'Tea':{'Without': 0.50, 'Normal': 0.60, 'Extra': 0.70 },
     }

order = items_dict[coffee][ingredients] * count

if coffee == 'Espresso' and count > 4:
    order *= 0.75

if ingredients == 'Without':
    order *=  0.65

if order > 15.0:
    order *= 0.80


print(f'You bought {count} cups of {coffee} for {order:.2f} lv.')


'''
Espresso
Without
10
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/3.CoffeMachine/3.CoffeMachine.js
function myFun(arg) {
  let coffee = arg[0]
  let ingredients  = arg[1]
  let count = +arg[2]

let obj ={
    'Espresso':{'Without': 0.90, 'Normal': 1.00, 'Extra': 1.20 },
    'Cappuccino':{'Without': 1.00, 'Normal': 1.20, 'Extra': 1.60 },
    'Tea':{'Without': 0.50, 'Normal': 0.60, 'Extra': 0.70 },
     }

let order = obj[coffee][ingredients] * count

  if (coffee === 'Espresso' && count > 4) {
    order -=  (order * 0.25)   
  }
    
  if (ingredients === 'Without' ) {
      order -=  (order * 0.35)   
     }

  if (order > 15.0) {
        order -=  (order * 0.20)   
      }

console.log(`You bought ${count} cups of ${coffee} for ${order.toFixed(2)} lv.`);
}

//myFun(['Espresso', 'Without', '10'])


  